docs/4_1_co-simulation_math.adoc

```diff
@@ -50,18 +50,19 @@ Dependent <<parameter,`parameters`>> (variables with <<causality>> = <<calculate
-It is also recommended but not mandatory that the function defined by the continuation of latexmath:[\mathbf{u}_{i, IU}] with latexmath:[\mathbf{u}_{i+1, IU}] be of smoothness latexmath:[C^{k}([t_i, t_{i+2}\])]  with the optional flag <<recommendedIntermediateInputSmoothness>> of value latexmath:[k]. +
-_[This can increase simulation speed for higher order multi-step solvers that in this case do not have to reset even at communication points.]_ +
-If an event happens at the communication point, <<continuous>> input variables may change discontinuously.
+Whether the <<input>> is a discrete-time or continuous-time variable is defined via attribute <<variability>> = <<discrete>> or <<continuous>> (see <<definition-of-model-variables>>).
+
 
 |latexmath:[\mathbf{u}_{i, IU}(t)]
 |Intermediate input variables corresponding to the input variables for intermediate update ("IU").
-The values of these variables are defined outside of the model.
-
+The values of these variables are defined outside of the model. +
 latexmath:[\mathbf{u}_{i, IU}] are functions defined on the interval latexmath:[ [t_i, t_{i+1}] ] with latexmath:[\mathbf{u}_{i, IU}(t_i)=\mathbf{u}(t_i)]. +
-With the optional flag <<recommendedIntermediateInputSmoothness>> with value latexmath:[k] FMUs can signal to the co-simulation algorithm that best convergence rates can be achieved if the functions are of the smoothness latexmath:[C^{k}([t_i, t_{i+1}\])], that is k-time continuously differentiable, with latexmath:[C^{0}] meaning continuous.
+For <<continuous>> input variables, if there is no discontinuity at time latexmath:[t_{i+1}] the importer must ensure that the input approximation function latexmath:[\mathbf{u}_{i, IU}] is consistent with the values of the input variable (latexmath:[\mathbf{u}_{i, IU}(t_{i+1})= \mathbf{u}(t_{i+1})]).
+It is also recommended that the function defined by the continuation of latexmath:[\mathbf{u}_{i, IU}] with latexmath:[\mathbf{u}_{i+1, IU}] be of smoothness latexmath:[C^{k}([t_i, t_{i+2}\])] with the optional flag <<recommendedIntermediateInputSmoothness>> of value latexmath:[k]. +
+_[This can increase simulation speed for higher order multi-step solvers that in this case do not have to reset at communication points.]_ +
+If there is a discontinuity in a <<continuous>> input variable at time latexmath:[t_{i+1}], then latexmath:[\mathbf{u}_{i, IU}] does not have to be consistent, but the importer must raise an event to signal the discontinuity. +
+FMUs can signal with the optional flag <<recommendedIntermediateInputSmoothness>> of value latexmath:[k] to the co-simulation algorithm that best convergence rates can be achieved if the functions are of smoothness latexmath:[C^{k}([t_i, t_{i+1}\])], that is k-time continuously differentiable, with latexmath:[C^{0}] meaning continuous.
```

Comment:
I would move this sentence before  line 61, because there we refer to <<recommendedIntermediateInputSmoothness>> for the continuation

Reply:
Done @chrbertsch !